modules/administration/pages/disconnected-setup.adoc

```diff
@@ -122,6 +122,33 @@ You can change SMT user settings in [path]``/etc/smt.conf``.
 
 
 
+== Mandatory Channels
+
+The corresponding {productname} Client Tools Channels are required, for {productname} to be able to synchronize a given channel.
+If these channels are not enabled, {productname} may fail to detect that product.
+
+Run the following command to enable these mandatory channels:
+
+SLES 12 and products based on it such as SLES for SAP or SLE HPC::
+RMT: [command]``rmt-cli products enable sle-manager-tools/12/x86_64``
++
+SMT: [command]``smt repos -p sle-manager-tools,12,x86_64``
+
+SLES 15 and products based on it such as SLES for SAP or SLE HPC::c
```

Comment:
Is the "c" after the `::` is a typo or a special format tag?

Reply:
no, it is a typo.  thanks for spotting!